rest/route/reliability_test.go

```diff
@@ -215,17 +243,21 @@ func TestParseValid(t *testing.T) {
 	assert.Equal([]string{"enterprise-rhel-62-64-bit", "enterprise-windows", "enterprise-rhel-80-64-bit"}, handler.filter.BuildVariants)
 	assert.Nil(handler.filter.Distros)
 	assert.Nil(handler.filter.StartAt)
-	assert.Equal(stats.GroupByDistro, handler.filter.GroupBy)            // default value
-	assert.Equal(stats.SortLatestFirst, handler.filter.Sort)             // default value
-	assert.Equal(reliabilityAPIMaxNumTasksLimit+1, handler.filter.Limit) // default value
+	assert.Equal(reliability.GroupByDistro, handler.filter.GroupBy) // default value
+	assert.Equal(reliability.SortLatestFirst, handler.filter.Sort)  // default value
+	assert.Equal(reliability.MaxQueryLimit, handler.filter.Limit)   // default value
 	assert.Equal(handler.filter.Significance, 0.1)
 
 }
 
 func TestParse(t *testing.T) {
 	assert := assert.New(t)
 
-	url := getURL(projectID, []string{"aggregation_expression_multiversion_fuzzer"})
+	url := getURL(projectID, map[string]interface{}{
+		"tasks":         "aggregation_expression_multiversion_fuzzer",
+		"after_date":    "2019-01-02",
+		"group_by_days": "10",
+	})
 	request, err := http.NewRequest("GET", url, bytes.NewReader(nil))
 	assert.NoError(err)
```

Comment:
I'd probably use require.NoError() in most cases rather than assert. 

(the APIs are the same, but require stops the execution of the test, whereas assert lets it continue, leaving the test susceptible to continue executing after invariant failure conditions.)

Reply:
ah, nice. I was unsure of the difference. require sounds like it provides more reasonable behaviour.

Reply:
My general rule of thumb is "use require most of the time, particularly when checking a condition that could cause later assertions to panic (e.g. errs and not-nil cases,) as well as assertions in larger loops that can be noisy." I think there's a place for assertions, when you have a smallish collection of related values that you want to verify, and if it were to break, you'd end up cycling through the test too many times to catch all the errors.

Reply:
Ok, then my changing everything was wrong. I'll change it to appropriate usage (and sorry for the github spam, I'll gather the remaining comments / responses into one)